docs/README.linux

```diff
@@ -190,7 +209,7 @@ in Settings->Videos->Player from "Auto Detect" to "VDPAU".
 6. Uninstalling
 -----------------------------------------------------------------------------
 
-Issue the commands prepending "sudo", if your user doesn't have write permission,
+Issue the commands pre-pending "sudo", if your user doesn't have write permission,
```

Comment:
prepending is correct, but there might be better ways of phrasing this grammatically